test/active_record/carrierwave_test.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+require_relative 'active_record_test_helper'
+
+if defined?(ActiveRecord)
```

Comment:
I wonder if we can run different tests in different environment setting, so that we don't have to checking if `ActiveRecord`, `Mongoid` are defined in each test files.

For example:
only run tests in `test/mongoid/` when using `gemfiles/mongoid_54.gemfile`
only run tests in `test/active_reocrd/` when using `gemfiles/active_record_51.gemfile`

I found that travis can run different script with different environment variables ([ref](https://gist.github.com/jamesarosen/e29076bd81a099f0f72e)). But did not know if it can detect different `gemfiles` or not.

```
script:
  - if [ "$TO_TEST" = "mongoid" ]; then bundle exec rake test_mongoid ; fi
  - if [ "$TO_TEST" = "active_record" ]; then bundle exec rake test_active_record; fi
```

Reply:
I thought of that too, let me do some research~